builtins/target-neon-i32x4.ll

```diff
@@ -258,6 +258,175 @@ define <WIDTH x double> @__sqrt_varying_double(<WIDTH x double>) nounwind readno
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; reductions
 
+dnl $1: scalar type
```

Comment:
I had to look up M4 docs to figure out what `dnl` is, we typically use just comments.
Though it's a matter of taste, I'm ok with it if you prefer it this way.